### DISCUSSION TITLE
MixedDataSource: Support live stream queries

### DISCUSSION THREAD

**Discussion by ZuperZee** [grafana/grafana#38309] [Dashboards, Panels & Visualizations]
**What happened**:

When switching from "-- Grafana --" to "-- Mixed --" datasource, the live stream stops.
![mixed-datasource-live-stream-bug](https://user-images.githubusercontent.com/39485579/122715524-4a025d00-d269-11eb-8703-4581a0e9bcf2.gif)

**What you expected to happen**:

The live stream should continue to update.

**How to reproduce it (as minimally and precisely as possible)**:

1. Create a new panel.
2. Select "-- Grafana --" as the datasource
3. Select "Live Measurements" as the query type
4. Select any channel
5. Switch to "-- Mixed --" datasource

**Environment**:
- Grafana version: 8.0.3
- Data source type & version: grafana
- OS Grafana is installed on: Ubuntu 20.10 (docker)
- User OS & Browser: Ubuntu 20.10 & Google Chrome
- Grafana plugins:
- Others:

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Reply by 0xMihir** [reply]
I don't think this is really a feature request, as this seems to be a bug with the way mixed data sources and live channels are handled

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
